tests/test_algorithm.py

```diff
@@ -4491,7 +4471,7 @@ def transactions_for_date(date):
             first_auto_close_transaction,
             {
                 'amount': -order_size,
-                'commission': 0.0,
+                'commission': None,
```

Comment:
When should commissions be `None` vs `0.0`?

Reply:
orders have commission, transactions do not.